From f121ce4d22ff23f16cc374c9a125d3815765bb0c Mon Sep 17 00:00:00 2001
From: Yaakov Selkowitz <yselkowitz@users.sourceforge.net>
Date: Mon, 22 Mar 2010 12:47:52 -0500
Subject: [PATCH 12/25] Disable setuid configure test on Cygwin

Only Xorg is installed setuid, so there is no need to run this configure
test on Cygwin.

Signed-off-by: Yaakov Selkowitz <yselkowitz@users.sourceforge.net>
(cherry picked from commit 24c9d48cd6999f1e90f9f0e2bbfebd8e95d314d8)
---
 configure.ac |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git xorg-server-1.8.0/configure.ac xorg-server-1.8.0/configure.ac
index 48cb724..1e65c51 100644
--- xorg-server-1.8.0/configure.ac
+++ xorg-server-1.8.0/configure.ac
@@ -673,6 +673,7 @@ AC_ARG_ENABLE(install-setuid,
 AC_MSG_CHECKING([to see if we can install the Xorg server as root])
 if test "x$SETUID" = "xauto" ; then
 	case $host_os in
+	    cygwin*)		SETUID="no"  ;;
 	    darwin*)		SETUID="no"  ;;
 	    *)
 	   	case $host_cpu in
-- 
1.7.0.4

